DPGAnalysis/Skims/python/ZMuSkim_cff.py

```diff
@@ -1,44 +1,166 @@
 import FWCore.ParameterSet.Config as cms
-
+from Configuration.StandardSequences.MagneticField_cff import *
 ### HLT filter
 import copy
 from HLTrigger.HLTfilters.hltHighLevel_cfi import *
+from  SimGeneral.HepPDTESSource.pythiapdt_cfi import *
```

Comment:
i'm not sure I see why this and the next several imports are needed? [I  do see why genericParticleProducer_cfi is needed]